index.js

```diff
@@ -24,6 +31,18 @@
 
 
 // ***** Question 3 *****
+let currentThirstLevel = 0
+
+const drinkWater = (currentThirstLevel) => {
+  if (currentThirstLevel >= 10) {
+    console.log('man I sure am thirsty')
+    currentThirstLevel -=1
+    console.log('ahh that hits the spot')
+    console.log(currentThirstLevel)
+  } 
+}
+drinkWater()
```

Comment:
Nice arrow function syntax here. A couple of notes for this one: 

- There's no need for the `currentThirstLevel` variable - the idea is just to pass it in when calling the function: `drinkWater(12)`
- Also no need for the `if` statement; there's nothing in the deliverable that says to make it only work if the value is greater or equal to 10
- Lastly, this function is meant to **return** the currentThirstLevel, not `console.log` it. 

Here's a working example:

```js
const drinkWater = (currentThirstLevel) => {
    console.log('man I sure am thirsty')
    currentThirstLevel -=1
    console.log('ahh that hits the spot')
    return currentThirstLevel
}
console.log(drinkWater(12))
```